### ISSUE TITLE
[semantic-issue] Add semantics for sklearn RandomizedSearchCV

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#172]
**Description**
Invokes the constructor to instantiate a RandomizedSearchCV object.
Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py

**Minimal running example**
Minimal running example using the semantics
```
from sklearn.model_selection import RandomizedSearchCV

tree_cv = RandomizedSearchCV(tree, param_grid, cv=5)
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.RandomizedSearchCV.html

**Note (optional)**
Returns an object of RandomizedSearchCV type.
